EarTrumpet/UI/Views/SettingsWindow.xaml

```diff
@@ -176,28 +179,30 @@
                 <CheckBox HorizontalAlignment="Left"
                           Content="{x:Static resx:Resources.SettingsUseLogarithmicVolume}"
                           IsChecked="{Binding UseLogarithmicVolume, Mode=TwoWay}" />
-                <!-- Localization required -->
-                <TextBlock VerticalAlignment="Center"
-                            Style="{StaticResource BodyText}"
-                            Text="Minimum dB in logarithmic slider" />
-                <StackPanel Orientation="Horizontal"
-                            Margin="20,0,0,0"
-                            HorizontalAlignment="Left"
-                            VerticalAlignment="Bottom">
+                <StackPanel Visibility="{Binding UseLogarithmicVolume, Converter={StaticResource BooleanToVisibilityConverter}}">
+                    <!-- Localization required -->
+                    <TextBlock VerticalAlignment="Center"
+                               Style="{StaticResource BodyText}"
+                               Text="Minimum dB in logarithmic slider" />
```

Comment:
Let's use `Logarithmic scale minimum`. If you'd like me to add the strings, let me know.